handbook/engineering/developer-insights/index.md

```diff
@@ -100,6 +100,27 @@ In addition to team specific channels, we communicate in the following org-wide
 
 [#developer-insights-org-leadership](https://sourcegraph.slack.com/archives/C01P1TVDJR4): This channel is for the Engineering Managers, Product Managers and Product Designers to discuss higher-level matters impacting the whole org.
 
+#### Acknowledgment
+
+In an async-first communication environment it is important to remove assumptions/uncertainty around whether teammates have seen, understood or acted on a message.  
+To assist in this regard we provide the following guideline for teammates to follow when communicating and responding in Slack. 
+
+**The most important thing to remember is not which emoji to use, but rather to remember to acknowledge and to do it in a clear and unambiguous way.**
+
+*When acknowledging a request:*
+
+- `:thumbsup:` (👍) = acknowledgment as in I see the request and will action it
+- `:white_check_mark:` (✅) = acknowledgment as in I have completed my action on the request
+
+*When acknowledging a statement:*
+
+- `:thumbsup:` 👍 = acknowledgment as in I agree with the statement
+- `:thumbsdown:` 👎 = acknowledgment as in I disagree with a statement - encouraged to always follow up with written response
```

Comment:
```suggestion
- `:thumbsup:` 👍 = I agree with the statement
- `:thumbsdown:` 👎 =  I disagree with a statement - encouraged to always follow up with written response
```

(I think "acknowledgement as in" is not necessary here and kind of a long phrase to keep repeating/rereading; if you agree then would remove in the above section as well)

Reply:
+1, I was going to leave the same feedback but couldn't figure out how to do it in a single suggestion :-)

Reply:
@Joelkw my copy savior 😉 
Yes let's go ahead with this.